docs/source/api/v5/types.rst

```diff
@@ -77,7 +77,7 @@ Response Structure
 	{ "response": [
 		{
 			"id": 48,
-			"lastUpdated": "2018-12-12 16:26:41+00",
+			"lastUpdated": "2018-12-12T20:49:28.111111Z"",
```

Comment:
missing time offset, like `+05:30`. This is how RFC format looks like: `2023-05-25T15:59:33.7096-06:00`

Reply:
RFC3339 uses `Z` to represent UTC time. It stands for "zero" because it can equivalently be written as `+00:00` or `-00:00`.

This is compliant, and you can see it in a bunch of other places as well.